cmd/tokengen/README.md

```diff
@@ -58,10 +58,10 @@ Usage:
   tokengen gen fabtoken [flags]
 
 Flags:
-  -a, --auditors strings   list of auditor keys in the form of <MSP-Dir>:<MSP-ID>
+  -a, --auditors strings   list of auditor keys in the form of <MSP-Dir>
```

Comment:
the description of this flag doesn't make sense anymore to me. How would I use it? Maybe something like: "list of auditor MSP directories containing the corresponding auditor keys"?

Reply:
same comment for issuers and tokengen gen dlog

Reply:
okay